packages/manager/cypress/e2e/databases/delete-database.spec.ts

```diff
@@ -2,124 +2,132 @@
-    mockDeleteDatabase(database.id, database.engine).as('deleteDatabase');
-
-    cy.visitWithLogin(`/databases/${database.engine}/${database.id}/settings`);
-    cy.wait('@getDatabase');
-
-    // Click "Delete Cluster" button.
-    ui.button
-      .findByAttribute('data-qa-settings-button', 'Delete Cluster')
-      .should('be.visible')
-      .click();
-
-    // Fill out type-to-confirm, click "Delete Cluster" button, confirm error.
-    ui.dialog
-      .findByTitle(`Delete Database Cluster ${database.label}`)
-      .should('be.visible')
-      .within(() => {
-        cy.findByLabelText('Cluster Name').click().type(database.label);
-
-        ui.buttonGroup
-          .findButtonByTitle('Delete Cluster')
-          .should('be.visible')
-          .should('be.enabled')
-          .click();
-
-        cy.wait('@deleteDatabase');
-      });
-
-    cy.url().should('endWith', '/databases');
-    ui.toast.assertMessage('Database Cluster deleted successfully.');
-  });
-
-  /*
-   * - Tests provisioning database deletion UI flow using mocked data.
-   * - Confirms that error is shown to user upon deletion attempt.
-   */
-  it('Cannot delete provisioning database clusters', () => {
-    const database = databaseFactory.build({
-      id: randomNumber(1, 1000),
-      type: 'g6-nanode-1',
-      label: randomLabel(),
-      region: randomItem(regions),
-      engine: 'mysql',
-      status: 'provisioning',
-      allow_list: [],
-      hosts: {
-        primary: undefined,
-        secondary: undefined,
-      },
-    });
-
-    const errorMessage =
-      'Your database is provisioning; please wait until provisioning is complete to perform this operation.';
-
-    mockGetDatabase(database).as('getDatabase');
-    mockDeleteProvisioningDatabase(
-      database.id,
-      database.engine,
-      errorMessage
-    ).as('deleteDatabase');
-
-    cy.visitWithLogin(`/databases/${database.engine}/${database.id}/settings`);
-    cy.wait('@getDatabase');
-
-    // Click "Delete Cluster" button.
-    ui.button
-      .findByAttribute('data-qa-settings-button', 'Delete Cluster')
-      .should('be.visible')
-      .click();
-
-    // Fill out type-to-confirm, click "Delete Cluster" button, confirm error.
-    ui.dialog
-      .findByTitle(`Delete Database Cluster ${database.label}`)
-      .should('be.visible')
-      .within(() => {
-        cy.findByLabelText('Cluster Name').click().type(database.label);
-
-        ui.buttonGroup
-          .findButtonByTitle('Delete Cluster')
-          .should('be.visible')
-          .should('be.enabled')
-          .click();
-
-        cy.wait('@deleteDatabase');
-        cy.findByText(errorMessage).should('be.visible');
+  databaseConfigurations.forEach(
+    (configuration: databaseClusterConfiguration) => {
+      it(`Deletes a ${configuration.linodeType} ${configuration.engine} v${configuration.version}.x ${configuration.clusterSize}-node cluster at ${configuration.region}`, () => {
```

Comment:
Similar to the previous comment, we just want to swap `it(...)` out with `describe(...)` here!

```suggestion
      describe(`Deletes a ${configuration.linodeType} ${configuration.engine} v${configuration.version}.x ${configuration.clusterSize}-node cluster at ${configuration.region}`, () => {
```